### PR TITLE
fix(cloudfoundry): add test for non-empty fields

### PR DESCRIPTION
**Pull request by cristhian-castaneda**

### CODE REVIEW

File: clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/model/v3/Process.java [hunk 1]
```diff
@@ -40,9 +41,7 @@ private HealthCheck() {}
 
     @Nullable private String type;
 
-    @JsonInclude(value = JsonInclude.Include.CUSTOM, valueFilter = HealthCheckData.class)
```
Comment (zachsmith1) [inline clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/model/v3/Process.java:43]: wait why did we remove this?

**Review by zachsmith1 — APPROVED** [review]
